js/viewer/viewer.js

```diff
@@ -36,6 +36,7 @@ function Viewer(parent, config) {
 
     // This boolean variable controls whether to show Channel names over the OSD or not.
     this._showChannelNamesOverlay = false;
+    this.prevAnnotation = null;
```

Comment:
Let's call this `_lastDrawnAnnotation`. `_` is supposed to emphasize that this is a "private" property. And comment why this is defined. 

Is there any other way that you can handle this? This looks out of place and ugly.